sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateHiveTableAsSelectCommand.scala

```diff
@@ -45,6 +46,19 @@ case class CreateHiveTableAsSelectCommand(
   override def innerChildren: Seq[LogicalPlan] = Seq(query)
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
+    // the CTAS's SELECT partition-outputs order should be consistent with
+    // tableDesc.partitionColumnNames
+    val partitionAttrs = tableDesc.partitionColumnNames.map {
+            p =>
```

Comment:
Could you please follow the coding style in https://github.com/databricks/scala-style-guide#spacing-and-indentation? Thanks!